### DISCUSSION TITLE
Logging Into Github on Terminal Permission Denied

### DISCUSSION THREAD

**Discussion by HiddenDragon01** [community/community#23889] [New to GitHub]
<p>Hello,</p>
<p>I changed my personal access token in keychain access, since I got the</p>
<p>remote: Support for password authentication was removed on August 13, 2021. Please use<br>
a personal access token instead. remote: Please see <a href="https://github.blog/2020-12-15-" rel="noopener nofollow ugc">https://github.blog/2020-12-15-</a><br>
token-authentication-requirements-for-git-operations/ for more information. fatal:<br>
unable to access “…” : The requested URL returned error: 403</p>
<p>message, however, after I entered my token in keychain access, I got:</p>
<p>remote: Permission to HiddenDragon01/Test.git denied to HiddenDragon01.<br>
fatal: unable to access ‘<a href="https://github.com/HiddenDragon01/Test.git/" rel="noopener nofollow ugc">https://github.com/HiddenDragon01/Test.git/</a>’: The requested URL returned error: 403</p>
<p>instead. I have no idea how to fix it - I have been trying for hours and nothing has worked. What should I do? Thank you</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Does your PAT have the right scopes? For pushing/fetching a repository it’ll usually need the <strong>repo</strong> scope, possibly <strong>workflow</strong> too if you use Actions.</p>

**Comment by HiddenDragon01** [comment]
<p>it worked, thank you!</p>

**Comment by sudo-nautilus** [comment]
<p>Um, I think - yes, I think,</p>
<p>If a comment or a reply helps you, you should mark it as the answer, in future, if someone encounters the same issue, they can easily find the solution by just looking into the marked comment. Simple as it sounds.</p>
